jobs/scheduler/spec

```diff
@@ -32,7 +32,7 @@ properties:
   scheduler.tls.client_scalingengine.cert:
```

Comment:
Let us get the naming correct.
scheduler.tls is redundant since these have nothing to do with scheduler's tls.

Follow conventions already set forth by Diego.
For example this should be:
scheduler.scalingengine.client_cert
scheduler.scalingengine.client_key
scheduler.scalingengine.ca_cert

We seem to have gone with a global CA concept which isn't good.